environment.yml

```diff
@@ -11,13 +11,12 @@ dependencies:
   - esmpy
   - esmvalcore>=2.1.0,<2.2
   - iris>=2.2.1,<3
-  - matplotlib>=3,<3.3
+  - matplotlib>3.3.1  # bug in 3.3.1 but 3.3.3 gets installed automatically
```

Comment:
Could you also pin cartopy to >=0.18 please? Cartopy < 0.18 is incompatible with matplotlib 3.3 and up, see https://github.com/SciTools/cartopy/issues/1615

Reply:
very good idea!!

Reply:
OK I ported the changes from #1893 here